### ISSUE TITLE
Feedback on v2

### ISSUE THREAD

**Issue by sayakpaul** [ayulockin/deepimageinpainting#2]
Great work @ayulockin! Here is some feedback:

- Clear the insallation outputs. 
- `if self.shuffle == True` can be ported to `if self.shuffle` since `shuffle` is of `bool` type.
- When you are creating the test generator you don't need to set `shuffle=True`. 
- The following code block needs more brief comments:

```python
def __data_generation(self, idxs):
    ## Initialize with empty arrays
    X_batch = np.empty((self.batch_size, self.dim[0], self.dim[1], self.n_channels)) # Masked image
    y_batch = np.empty((self.batch_size, self.dim[0], self.dim[1], self.n_channels)) # Original image

    ## Iterate through random indexes
    for i, idx in enumerate(idxs):
      tmp_image = self.X[idx].copy()
      
      ## Masked image
      res = self.__createMask(tmp_image)
      
      X_batch[i,] = res/255
      y_batch[i] = self.y[idx]/255
      
    return X_batch, y_batch
```
For example, you could put a comment just before you are parsing the masks and setting the (X,y) pairs for the given batch. 

- `res` could be given a more descriptive name. Example: `masked_image_batch`.
- After you are showing a set of masked images, it might be a good idea to show a couple of image -> image pairs. 
- You can remove this one:
```python
def __init__(self):
    pass
```

- Break the line of code which is long. 
- The following block is a bit hard to follow (I believe the essence of a U-Net like architecture remains here):
```python
up = keras.layers.concatenate([keras.layers.Conv2DTranspose(filters=up_filters, kernel_size=up_kernel, strides=up_stride, padding=padding)(conv), shared_layer], axis=3)
```
